packages/manager/modules/pci/src/projects/project/notebooks/dashboard/dashboard.routing.js

```diff
@@ -9,8 +10,17 @@ export default /* @ngInject */ ($stateProvider) => {
         $state.href($state.current.name, $transition$.params()),
       notebookId: /* @ngInject */ ($transition$) =>
         $transition$.params().notebookId,
-      notebook: /* @ngInject */ (notebookId, notebooks) =>
-        find(notebooks, { id: notebookId }),
+      notebook: /* @ngInject */ (
+        notebookId,
+        notebooks,
+        coreConfig,
+        CucRegionService,
+      ) =>
+        new Notebook(
```

Comment:
Why do you need to create a new object here? If is it `coreConfig` and `CucRegionService`, cant we pass it in the top level route itself?
And I don't feel it is right to pass `coreConfig` and `CucRegionService` (which are angular services) into a native class...

Reply:
Now i got the notebook via the list.